PWGMM/Mult/Tasks/dndeta-hi.cxx

```diff
@@ -9,12 +9,27 @@
 // granted to it by virtue of its status as an Intergovernmental Organization
 // or submit itself to any jurisdiction.
 
+#include <array>
```

Comment:
In lines 12-16, none of this system header is actually used in your code with a possible exception of `cmath`.